app/html/src/server/framework-preset-html.ts

```diff
@@ -1,4 +1,7 @@
-export function webpack(config) {
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Configuration } from 'webpack';
```

Comment:
I don't know well how this part of SB is working but maybe `webpack` should be a `peerDependency`? 
 @ndelangen can surely help us with this ;)

Reply:
You can create a dependency on webpack for now, in monoconfig, I'll try an consolidate these types into a coherent whole.

When you add the dependency, be sure to add a wide version range.

Reply:
I think we should export the preset function types from `@storybook/core`

Reply:
Just to clarify, in this case `Configuration` is being picked up through `@types/webpack` which is a devDependency at the root of the repo. Technically, there's no runtime use or type definitions coming from `webpack` here.

It would make sense to add `@types/webpack` as a devDependency here—I could match the version at the root of the repo to avoid any conflicts. Given the above, do you guys still feel I should add `webpack` as a dependency? Let me know your thoughts 😄

Reply:
@gaetanmaisse @ndelangen let me know your thoughts on my last comment and I'll make the change :)